src/database/DataTypes/Sensor.js

```diff
@@ -42,6 +42,10 @@ export class Sensor extends Realm.Object {
   get mostRecentTemperatureBreach() {
     return this.location?.mostRecentTemperatureBreach;
   }
+
+  get batteryLevelString() {
+    return `${this.batteryLevel}%`;
+  }
```

Comment:
just to make the UI easier. Maybe shouldn't do this, I dunno

Reply:
I think does the job for now. I think longer term, maybe suitable to add as part of parsing and formatting utilities refactoring?